README.md

```diff
@@ -146,17 +146,13 @@ How to run a "hello-world" plugin the Cloudflare way:
 2. Run `bun plugin:hello-world` to spin up a local server for the "hello-world" plugin
 3. Update the bot's config file in the repository where you use the bot (`OWNER/REPOSITORY/.github/.ubiquibot-config.yml`):
 
-```
+```yml
 plugins:
-  'issue_comment.created':
-    - name: "hello-world-plugin name"
-      description: "hello-world-plugin description"
-      command: "/hello"
-      example: "/hello example"
-      skipBotEvents: true
```

Comment:
@gentlementlegen Aren't `name, description, command, example` required properties?

Reply:
I think they are read from the manifest of each plugin now

Reply:
Correct, they are now located inside the manifest so plugins are responsible to provide these informations themselves.